src/core/lib/event_engine/default_event_engine.cc

```diff
@@ -78,5 +81,18 @@ void ResetDefaultEventEngine() {
   g_event_engine->reset();
 }
 
+namespace {
+grpc_core::ChannelArgs EnsureEventEngineInChannelArgs(
+    grpc_core::ChannelArgs args) {
+  if (args.ContainsObject<EventEngine>()) return args;
+  return args.SetObject<EventEngine>(GetDefaultEventEngine());
+}
+}  // namespace
+
+void RegisterEventEngine(grpc_core::CoreConfiguration::Builder* builder) {
```

Comment:
Suggest calling this `RegisterEventEngineChannelArgPreconditioning()`, to make it clear what it's actually registering.

Reply:
Done